Sources/Remote Logging/Crash Logging/SentryClient+Shim.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SentryClient+Shim.swift
+//  AutomatticTracksiOS
+//
+//  Created by BJ Homer on 7/27/21.
+//
+
+import Foundation
```

Comment:
I haven't tried to build this, but just as a quick checked I'd say this import is not necessary?

Reply:
I didn't notice that call. That's what I get for leaving comments after having only "quick checked" the code 😅